README.md

```diff
@@ -302,3 +302,21 @@ startKeyUX(window, [
   hiddenKeyUX()
 ])
 ```
+
+
+### Mac Compatibility Mode
+
+It's common to use the Meta (<kbd>⌘</kbd>) modifier for hotkeys on Mac, while
+Window and Linux usually favor the Ctrl key. To provide familiar experience on
+all platforms, enable the Mac compatibility mode:
+
+```js
+import { hotkeyKeyUX, startKeyUX, MAC_COMPAT } from 'keyux'
+
+startKeyUX(window, [
+  hotkeyKeyUX({ ...MAC_COMPAT })
```

Comment:
I do not like this API, since we are taking `applyCompat` code always even for people who don’t need this feature.

Let’s change API to:

```js
hotkeyKeyUX(overrides, transformers)
```

and `macCompat` will be one of possible `transformers`.

Reply:
Looks good, done.

It's now tempting to implement overrides as tranformers as well :)